sickbeard/name_parser/parser.py

```diff
@@ -568,6 +619,10 @@ def is_anime(self):
             return True
         return False
 
+    @property
+    def is_proper(self):
+        return re.search(r'(^|[\. _-])(proper|repack)([\. _-]|$)', self.extra_info, re.I) is not None if self.extra_info else False
```

Comment:
Should just check if "if guess.get('proper_count', 0) > 0" , outherwise it doesn't detect `ReRip`  and `Real`